### DISCUSSION TITLE
Possible to use a standard Bind Mount in future?

### DISCUSSION THREAD

**Discussion by Jacksaur** [goniszewski/grimoire#142] [Ideas]
This is more of just an OCD irritation that anything major usability wise: Will it be possible to use Grimoire under a standard Bind mount to a folder in future, rather than having to create a separate Docker volume just for it?
All my containers just write directly into their own appdata subfolders currently, which makes it very easy to back up and restore them with a single Borg script I run.

**Comment by goniszewski** [comment]
Hello @Jacksaur, sorry for the late reply!

You can modify your compose file to do just that by changing `- grimoire_data:/app/data/` to, e.g. `- ./data:/app/data`. Or do you have something else in mind?

**Comment by Jacksaur** [comment]
Ooh, it seems I may have misjudged what was happening then. It's been a while since I tried (Sorry for taking so long to reply myself, too!), but I vaguely remember trying in the past and getting an error while doing so. I must have mistakenly associated the error with moving from the Docker Volume to a bind mount.

The error I get is:
```
$ bun run ./migrate
1 | import { Database } from 'bun:sqlite';
2 | import { drizzle } from 'drizzle-orm/bun-sqlite';
3 | import { migrate } from 'drizzle-orm/bun-sqlite/migrator';
4 |
5 | const sqlite = new Database('data/db.sqlite');
                   ^
SQLiteError: unable to open database file
 errno: 14
  code: "SQLITE_CANTOPEN"
      at new Database (bun:sqlite:178:14)
      at /app/migrate.js:5:16
Bun v1.1.29 (Linux x64 baseline)
error: script "run-migrations" exited with code 1
```
This is from a fresh install, with nothing to migrate from in the first place. Should I submit this as its own Issue?

**Comment by Jacksaur** [comment]
Seems it was caused by switching to a bind mount after all, and you've already acknowledged it under #133 
Apologies for cluttering your discussions like this, I should have done more research on my part :V
